python/taichi/lang/common_ops.py

```diff
@@ -0,0 +1,56 @@
+class TaichiOperations:
+    def __neg__(self):
+        import taichi as ti
```

Comment:
I've actually long been confused by this pattern (not related to this PR, as I see this in other places as well). What's wrong if we just `import taichi as ti` once at the beginning?

Reply:
See https://stackabuse.com/python-circular-imports/
If we move this outside, taichi maybe imported before taichi.lang.ops were imported, and we may get an NameError for ti.neg, depends on the importing order. (actually pretty like C++ #include and #pragma once)

Reply:
Ah thanks! (BTW, I guess it's better to decompose the Python lib into smaller sub modules as well...)